testnet2/testnet2.toml

```diff
@@ -6,28 +6,28 @@ Key = "network"
 Value = "testnet2"
 
 [API]
-
   [API.GRPC]
     Hosts = [
       "api-n00.validators-testnet.vega.rocks:3007",
       "api-n02.validators-testnet.vega.rocks:3007",
       "api-n03.validators-testnet.vega.rocks:3007",
       "api-n04.validators-testnet.vega.rocks:3007",
       "api-n05.validators-testnet.vega.rocks:3007",
-      "vega-testnet.anyvalid.com:3007"
+      "vega-testnet.anyvalid.com:3007",
+      "testnet.grpc.vega.xprv.io:443"
     ]
     Retries = 5
-
   [API.GraphQL]
     Hosts = [
       "https://api-validators-testnet.vega.rocks/graphql",
-      "https://vega-testnet.anyvalid.com/query"
+      "https://vega-testnet.anyvalid.com/query",
+      "https://testnet.vega.xprv.io/datanode/query"

```

Comment:
When I use this URL on https://validator-testnet.console.vega.xyz, it doesn't work. However, I don't see any apparent reason for it not working. I am asking our team to help investigate this.